doc/data/messages/t/too-many-lines/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
Details should contain a rational for why file with too much lines are bad.

Reply:
Added some words to details.rst